<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientConnectionInfoDialog</class>
 <widget class="QDialog" name="ClientConnectionInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 23px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Current Connection Role: Client</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 23px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
